App.js

```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
can you put this in a file of its own and put it inside `lib` folder.
The full path will be `src/lib/request.js`